chrome/content/zotero/standalone/standalone.js

```diff
@@ -715,6 +715,23 @@ const ZoteroStandalone = new function() {
 			);
 			return { remove: result === 0, report: null };
 		};
+		// A11y: when a popup appears, mark which buttons are checked for screen readers
+		doc.addEventListener("shown", (event) => {
+			for (let item of [...event.target.querySelectorAll("panel-item")]) {
+				item.button.setAttribute("role", "menuitemcheckbox");
+				item.button.setAttribute("aria-checked", item.checked);
+			}
+		}, true);
+
+		// A11y: after a click, check if the panel with plugin details appeared.
+		// If so, delete a misleading role=tabpanel
```

Comment:
Should clarify that we're hiding the tab bar in fetch_xulrunner — that's why it's not a tab.

Reply:
Oh, yeah. I actually didn't realize that it's explicitly hidden [here](https://github.com/zotero/zotero/blob/d879c507a55bd7060699a943a26554f633fd24a4/app/scripts/fetch_xulrunner#L290). Good to know, I'll add a comment.